src/widgets/cards/summary-card.component.tsx

```diff
@@ -53,13 +54,22 @@ export default function SummaryCard(props: SummaryCardProps) {
   }
 }
 
+function showComponent(component, name): void {
+  newWorkspaceItem({
+    component: component,
+    name: name,
+    props: { match: { params: {} } },
```

Comment:
we should refactor all components that need the `match` prop to `useRouteMatch()` hook and stop prop plumbing `match` into all of them. Cc @jonathandick

Reply:
Refactored to use `useRouteMatch`